app/src/main/java/com/yacgroup/yacguide/database/tourbook/TourbookExporter.kt

```diff
@@ -41,11 +49,57 @@ class TourbookExporter(
     private val _partnersKey = "partners"
     private val _notesKey = "notes"
 
+    enum class ExportFormat {
+        eJSON,
+        eJSONVEROBSE,
+        eCSV;
```

Comment:
Unnecessary `;`

Reply:
What is the alternative? I don't want to use Strings.

Reply:
I don't mean the enum itself but only the semicolon